crates/runestick/src/modules/char.rs

```diff
@@ -1,13 +1,25 @@
 //! The `std::char` module.
 
-use crate::{ContextError, Module};
+use crate::{ContextError, Module, Value, VmError, VmErrorKind};
 use std::char::ParseCharError;
 
 /// Construct the `std::char` module.
 pub fn module() -> Result<Module, ContextError> {
     let mut module = Module::with_crate_item("std", &["char"]);
     module.ty::<ParseCharError>()?;
+    module.function(&["from_int"], char_from_int_impl)?;
     Ok(module)
 }
 
+fn char_from_int_impl(value: Value) -> Result<Option<Value>, VmError> {
+    let inner: i64 = value.into_integer()?;
```

Comment:
Rune can do this conversion if you use `i64` as an argument directly. Or `value: i64`.

Reply:
Ah, yes. I wanted to capture conversion errors but doesn't matter here.